fuse/natchitoches_so.py

```diff
@@ -0,0 +1,59 @@
+import deba
+from lib.columns import (
+    rearrange_allegation_columns,
+)
+from lib import events
+from lib.personnel import fuse_personnel
+from lib.post import load_for_agency
+import pandas as pd
+
+
+def fuse_events(cprr, post):
+    builder = events.Builder()
+    builder.extract_events(
+        cprr,
+        {
+            events.COMPLAINT_RECEIVE: {
+                "prefix": "receive",
+                "keep": ["uid", "agency", "allegation_uid"],
+            }
+        },
+        ["uid", "allegation_uid"],
+    )
+    builder.extract_events(
+        post,
+        {
+            events.OFFICER_LEVEL_1_CERT: {
+                "prefix": "level_1_cert",
+                "parse_date": "%Y-%m-%d",
+                "keep": ["uid", "agency", "employment_status"],
+            },
+            events.OFFICER_HIRE: {
+                "prefix": "hire",
+                "keep": ["uid", "agency", "employment_status"],
+            },
+            events.OFFICER_PC_12_QUALIFICATION: {
+                "prefix": "last_pc_12_qualification",
+                "parse_date": "%Y-%m-%d",
+                "keep": [
+                    "uid",
+                    "agency",
+                    "employment_status",
+                ],
+            },
+        },
+        ["uid"],
+    )
+    return builder.to_frame()
+
+
+if __name__ == "__main__":
+    cprr = pd.read_csv(deba.data("match/cprr_natchitoches_so_2018_2021.csv"))
+    agency = cprr.agency[0]
+    post = load_for_agency(agency)
+    per_df = fuse_personnel(cprr, post)
+    com_df = rearrange_allegation_columns(cprr)
+    event_df = fuse_events(cprr, post)
+    event_df.to_csv(deba.data("fuse/event_natchitoches_so.csv"), index=False)
+    com_df.to_csv(deba.data("fuse/com_natchitoches_so.csv"), index=False)
+    per_df.to_csv(deba.data("fuse/per_natchitoches_so.csv"), index=False)
```

Comment:
This script `fuse/natchitoches_so.py` does not appear to run. Please check that the file `.deba/deps/fuse.d` list this script.

Reply:
It wasn't there before. Confused as to why. It is currently there. 

```
$(DEBA_DATA_DIR)/fuse/event_natchitoches_so.csv $(DEBA_DATA_DIR)/fuse/com_natchitoches_so.csv $(DEBA_DATA_DIR)/fuse/per_natchitoches_so.csv &: $(DEBA_MD5_DIR)/fuse/natchitoches_so.py.md5 $(DEBA_DATA_DIR)/match/cprr_natchitoches_so_2018_2021.csv $(DEBA_DATA_DIR)/clean/pprr_post_2020_11_06.csv $(DEBA_MD5_DIR)/datavalid.yml.md5 | $(DEBA_DATA_DIR)/fuse
	$(call deba_execute,fuse/natchitoches_so.py)
```